.gitignore

```diff
@@ -1,3 +1,5 @@
+*.sw[a-z]
+*.un~
```

Comment:
Can you put these entries in either `.git/info/exclude` or your global `.gitignore` please? I prefer to keep editor specific stuff out of repository .gitignores.